src/NuGet.Core/NuGet.Commands/RestoreCommand/ProjectRestoreCommand.cs

```diff
@@ -75,8 +75,11 @@ public ProjectRestoreCommand(ILogger logger, ProjectRestoreRequest request)
                 _request.MaxDegreeOfConcurrency,
                 token);
 
+            // The user global package folder is always ordered first.
+            var userPackageFolder = localRepositories.First();
```

Comment:
Yes, it seems like we do treat the user package folder different, so a type that models this would be good.

---
In reply to: [66690103](https://github.com/NuGet/NuGet.Client/pull/673#discussion_r66690103) [](ancestors = 66690103)